cascades-server/domain/logic/src/main/java/pl/gov/coi/cascades/server/domain/launchdatabase/UsernameAndPasswordCredentialsGeneratorService.java

```diff
@@ -31,22 +31,21 @@ public UsernameAndPasswordCredentialsGeneratorService() {
      */
     public UsernameAndPasswordCredentials generate() {
         String username = generateUsername();
-        char[] password = generatePassword();
+        char[] password = generateCharArrayPassword();
         return new UsernameAndPasswordCredentialsImpl(username, password);
     }
 
     private static String generateUsername() {
-        String uuid = UUID.randomUUID().toString();
-        return uuid.substring(0, USERNAME_LENGTH);
+        return RandomStringUtils.randomAlphabetic(USERNAME_LENGTH);
```

Comment:
`RandomStringUtils` should not be used as you don't control `Random` that it uses.